doc/enterprise-shield/p_enterprise_shield_cluster.md

```diff
@@ -0,0 +1,132 @@
+-   [Home](../../index.md)
+-   [Documentation](../index.md)
+-   Security with KAAZING Gateway
+
+Walkthrough: Configure Enterprise Shield™ for High Availability ![This feature is available in KAAZING Gateway - Enterprise Edition](../images/enterprise-feature.png)
+=================================================================
+
+This step extends the use case to configure an Enterprise Shield™ configuration described in [Walkthrough: Configure Enterprise Shield](p_enterprise_shield_config.md) into a cluster configuration that has two Gateways running in the DMZ and a matching set of Gateways in the trusted network. This configuration provides maximum security and also high availability because it pairs each internal Gateway to one explicit cluster member in the DMZ to provide failover (system resilience) during hardware or network failures. The complete configuration files for the cluster are available on `kaazing.org` at `enterprise-shield-use-case-3-cluster.xml`.
```

Comment:
> This configuration provides maximum security and also high availability because it pairs each internal Gateway to one explicit cluster member in the DMZ to provide failover (system resilience) during hardware or network failures.

Lets not emphasize the pairing here. This will go away once we have cluster-to-cluster communication, and we want to minimize future changes to the doc when that becomes available.

How about:

> This configuration provides maximum security and also high availability for failover (system resilience) during hardware or network failures.